Tests/FluentAssertions.Specs/Primitives/TimeOnlyAssertionSpecs.cs

```diff
@@ -0,0 +1,861 @@
+using System;
+using Xunit;
+using Xunit.Sdk;
+
+#if NET6_0_OR_GREATER
+
+namespace FluentAssertions.Specs.Primitives
+{
+    public class TimeOnlyAssertionSpecs
+    {
+        [Fact]
+        public void Should_succeed_when_asserting_nullable_timeonly_value_with_value_to_have_a_value()
```

Comment:
♻️ (refactor) Please group the tests by API so you don't have to repeat the action you're doing.